cfme/configure/access_control.py

```diff
@@ -262,6 +262,7 @@ def _retrieve_ext_auth_user_groups(self):
 
     def add_group_from_ldap_lookup(self):
         self._retrieve_ldap_user_groups()
+        sel.wait_for_ajax()
```

Comment:
I'd say these tests need to be completely rewritten and according to dajo mshriver is going to do that soon
So, I added as easy fix as it is possible